tests/securedrop/install_workstation.pm

```diff
@@ -16,6 +16,55 @@ use strict;
 use testapi;
 use networking;
 
+sub install_staging {
+    # Assumes terminal window is open
+
+    # NOTE: These are done via qvm-run instead of gnome-terminal so that we
+    # can know in case they failed.
+    assert_script_run('qvm-run -p work -- gpg --keyserver hkps://keys.openpgp.org --recv-key "2359 E653 8C06 13E6 5295 5E6C 188E DD3B 7B22 E6A3"');
+    assert_script_run('qvm-run -p work -- "gpg --armor --export 2359E6538C0613E652955E6C188EDD3B7B22E6A3 > securedrop-release-key.pub"');
+    assert_script_run('qvm-run -p work -- sudo rpmkeys --import securedrop-release-key.pub');
+    assert_script_run('qvm-run -p work -- "echo -e \"[sd]\nenabled=1\nbaseurl=https://yum-qa.securedrop.org/workstation/dom0/f37\nname=boostrap\"  | sudo tee /etc/yum.repos.d/securedrop-temp.repo"');
+    assert_script_run('qvm-run -p work -- dnf download -y securedrop-workstation-dom0-config');
+    assert_script_run('qvm-run -p work -- "rpm -Kv securedrop-workstation-dom0-config-*.rpm"');  # TODO confirm output is correct
+    assert_script_run('qvm-run -p work -- "cat /home/user/securedrop-workstation-dom0-config-*.rpm" > securedrop-workstation.rpm');
+    assert_script_run('sudo dnf -y install securedrop-workstation.rpm');
+};
+
+sub install_dev {
+    # Assumes terminal window is open
+
+
+    assert_script_run('qvm-check sd-dev || qvm-create --label gray sd-dev --class StandaloneVM --template debian-12-xfce');
+
+    # Building SecureDrop Workstation RPM and installing it in dom0
+    assert_script_run('qvm-run -p sd-dev "git clone https://github.com/freedomofpress/securedrop-workstation"');
+    assert_script_run('qvm-run -p sd-dev "git -C securedrop-workstation checkout ' . get_var('GIT_REF') . '"');
+
+    # SecureDrop dev. env. according to https://developers.securedrop.org/en/latest/setup_development.html
+    # DOCKER INSTALL according to https://docs.docker.com/engine/install/debian/
+    assert_script_run('qvm-run -p sd-dev "sudo apt-get update"');
+    assert_script_run('qvm-run -p sd-dev "sudo apt-get install -y ca-certificates curl"');
+    assert_script_run('qvm-run -p sd-dev "sudo install -m 0755 -d /etc/apt/keyrings"');
```

Comment:
Once this lands I'll send a PR to switch this over to podman, which should be faster since we won't have to set up the custom APT repo nor restart the VM.